scripts/email/lib.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, workspace_dir: str | Path, own_email: str | None = None):
         Args:
             workspace_dir: Path to the workspace directory
             own_email: The agent's own email address (used to avoid replying to self).
-                      If None, will use AGENT_EMAIL environment variable or default to "bob@superuserlabs.org"
+                      If None, will use AGENT_EMAIL environment variable or default to "agent@example.org"
```

Comment:
I don't think it should default to a bad email like this, if no valid email is configured then that seems like it should be a fatal misconfiguration (outside of testing)